README.md

```diff
@@ -100,28 +115,24 @@ const chain = new SetupChain({
   }
 })
 
-await chain.execute()
-{
-  b: {
-    c: 2
-  , d: {
-      e: 3
-    }
-  }
-}
+const state = await chain.execute()
```

Comment:
nit: `state2` or `initial_state` or something. same with `chain1`/`chain`. just something to clarify they are separate examples.

Reply:
Ah, yep. That was an incomplete thought.